catkin_ws/src/image_rotation/include/image_rotation/ImageAlgorithm.h

```diff
@@ -0,0 +1,17 @@
+#ifndef IMAGE_ALGORITHM_H
```

Comment:
files should be named with snake case words separated by underscores

Reply:
fixed.